scaladoc/src/dotty/tools/scaladoc/Scaladoc.scala

```diff
@@ -207,7 +207,7 @@ object Scaladoc:
         classpath.get,
         bootclasspath.get,
         destFile,
-        siteRoot.nonDefault,
+        Option(siteRoot.nonDefault.getOrElse(siteRoot.default)),
```

Comment:
I actually think if we wanted to do this we could instead do

```scala
Option(siteRoot.withDefault(siteRoot.default))
```

However, this causes a _ton_ of warnings when you do a publish local saying that it can't resolve a bunch of links. Could you dive into that and see why that's happening?

Reply:
Isn't that equivalent to just `Option(siteRoot.get)`?